tests/property/test_tally.py

```diff
@@ -131,76 +131,64 @@ def test_tally_ballot_invalid_input_fails(
                 from_ciphertext_ballot(encrypted_ballot, BallotBoxState.CAST),
             )
 
-        subject = CiphertextTally("my-tally", metadata, context)
```

Comment:
do we want to move away from this pattern permanently?  I generally like to use generic names for local vars inside tests, things like `subject` and `data` and `result` or `expected` and `actual` - things like that - but i could be convinced its better to use descriptive names.  What I like about identifying the `subject` specifically is that no matter where you are in a code block the object under test is always clear.

Reply:
I think when tests are centered a bit better it's useful but I think there is higher usability in describing what we are doing.